### DISCUSSION TITLE
Poly/Mono mode

### DISCUSSION THREAD

**Discussion by BobanSpasic** [probonopd/MiniDexed#261] [Q&A] [answered]
Hi.
Could someone, please, implement the Mono play mode? 
The settings should go under the Performance menu (like on DX7).

**Comment by probonopd** [comment, marked as answer]
Hello @BobanSpasic.

Looks like @dcoredump has implemented Mono mode in https://codeberg.org/dcoredump/Synth_Dexed/:

```
void setMonoMode(bool mode);
bool getMonoMode(void);
```

So we would "just" need to add a menu option for it, and save it in `performance.ini`.

Perhaps @arsamus could have a look at it?
